chroma-manager/storage_server.repo

```diff
@@ -11,13 +11,13 @@ enabled_metadata=1
 
 [lustre]
 name=Lustre Server
-baseurl=https://build.whamcloud.com/lustre-b2_10_last_successful_server/
+baseurl=https://build.hpdd.intel.com/jobs-pub/lustre-b2_10/configurations/axis-arch/x86_64/axis-build_type/server/axis-distro/el7/axis-ib_stack/inkernel/builds/26/archive/artifacts/
```

Comment:
We need to keep using the _whamcloud.com_ domain because of proxy issues inside of Intel.

Inside of Intel, proxy configurations exempt _*.intel.com_ from needing to use the proxy but _hpdd.intel.com_ is outside of the proxy and so actually needs to use the proxy.

There is no way to make exemptions from the exemptions so as to not proxy for _*.intel.com_ but proxy for _hpdd.intel.com_.  The only unfortunate alternative is to list all of the internal _.intel.com_ subdomains (except _hpdd.intel.com_ of course) in the proxy exemption list and play whack-a-mole every time something new is discovered not working because it needs adding to the list.